easybuild/easyconfigs/d/dbus-glib/dbus-glib-0.108-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'dbus-glib'
+version = '0.108'
+
+homepage = 'http://dbus.freedesktop.org/doc/dbus-glib'
+description = """D-Bus is a message bus system, a simple way for applications to talk to one another."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://dbus.freedesktop.org/releases/dbus-glib']
+sources = [SOURCE_TAR_GZ]
+checksums = ['9f340c7e2352e9cdf113893ca77ca9075d9f8d5e81476bf2bf361099383c602c']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('GLib', '2.53.5'),
+    ('DBus', '1.12.2'),
+    ('expat', '2.2.4'),
```

Comment:
@vanzod Since we're stuck between a rock & a hard place here with `expat`, can we make it a build dependency here somehow, for example by linking it statically?

Reply:
@vanzod Any updates on this?

Reply:
@boegel I tried but all attempts of building it statically failed. How do you want to proceed?

Reply:
@vanzod Well, we shouldn't be introducing conflicts, not sure how to proceed...